src/geom/Point.cpp

```diff
@@ -41,6 +41,7 @@ using namespace std;
 namespace geos {
 namespace geom { // geos::geom
 
+const static FixedSizeCoordinateSequence<0> emptyCoords;
```

Comment:
Its `getDimension()` always returns `3`. But 2D and 3D empty points are the different things :slightly_smiling_face:

Reply:
I guess the 3D one is even emptier than the 2D one. Should be fixed now.